code/modules/events/immovable_rod.dm

```diff
@@ -15,8 +15,8 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 
 /datum/event/immovable_rod/start()
 	var/startside = pick(GLOB.cardinal)
-	var/turf/startT = spaceDebrisStartLoc(startside, 1)
-	var/turf/endT = spaceDebrisFinishLoc(startside, 1)
+	var/turf/startT = spaceDebrisStartLoc(startside, 2)
```

Comment:
Do we have to hardcode the station z-level in here? Isn't there some way to just get 'the station's z level'?